tests/lean/lcnfTypes.lean

```diff
@@ -119,6 +119,8 @@ def testMono (declName : Name) : MetaM Unit := do
   let mono ← LCNF.toMonoType base
   IO.println s!"{declName} : {← ppExpr mono}"
 
+-- Universe levels are erased, and when the delaborator sees type incorrect applications it always inserts a `@`.
+-- This is why one sees `@List Ty` rather than `List Ty` in the following tests.
```

Comment:
Is there anything we could do against this?

Reply:
Ah, there is an explicit `pp.explicit` in the test, so I suppose this regression is expected

Reply:
It turns out that `match` missing pattern errors also lack universe levels, so this helps with that as well.